### ISSUE TITLE
PDF: Button label change based on the application state

### ISSUE THREAD

**Issue by kcullion** [icgc-argo/dac-ui#181]
This pdf button label needs to be changed based on application state
![image2021-5-21_11-58-14](https://user-images.githubusercontent.com/18424138/124789357-3701b500-df18-11eb-8e67-32a61b827b7c.png)

- [x] In DRAFT and REVISIONS REQUESTED state - button label is `DRAFT PDF` (FOR TESTING: Clicking this button downloads a non-finalized, watermarked application pdf)
- [x] In SIGN & SUBMIT state - button label is FINALIZED PDF (FOR TESTING: Clicking this button downloads a finalized, non-watermarked application pdf)
- [ ] DACO REVIEW and APPROVED - button label is SIGNED PDF (FOR TESTING: Clicking this button downloads the signed application pdf. This is what the applicant has signed and submitted.If an ethics letter exists, it will be zipped with the signed pdf)

**ALSO FIX**
- [x] the loading state for the PDF download should not resize the button (this happens because the button text changes)

**Comment by kcullion** [comment]
Tested on July 9. All happening as expected!

- button loading state looks good!
- I see draft pdf label when in draft state
- when I move to sign and submit state, I see button label change to finalized pdf and it downloads the non-watermarked pdf
- when I move the application to the "DACO REVIEW" state, I see the label change to "Signed pdf". @rosibaj how do we test that it will download the signed copy that they uploaded to submit?
- When I moved the application to the "REVISIONS REQUESTED" state, I see the label as "Draft pdf" and it downloads the watermark pdf
- When I moved application to the "APPROVED" state, I see the label as "Signed pdf" - just not if it downloads their uploaded signed version?
